cds_ils/migrator/utils.py

```diff
@@ -153,6 +153,16 @@ def get_acq_ill_notes(record):
     item_info = record.get("item_info")
     if item_info:
         result = "item info: {0}\n\n".format(item_info.strip())
+        try:
+            result = (
+                result.encode("ascii")
+                .decode("unicode-escape")
+                .encode("iso-8859-1")
```

Comment:
why do you use so many encode-decode ? especially why `iso-8859-1`  ?

Reply:
See [here](https://stackoverflow.com/questions/49756071/python-3-6-utf-8-to-unicode-conversion-string-with-double-backslashes/49756591)